doc/tutorials/rt_industry.rst

```diff
@@ -0,0 +1,456 @@
+      Copy /usr/lib/acrn/acrn.efi to /mnt/EFI/acrn
+      Check ACRN efi boot event
+      Clean all ACRN efi boot event
+      Check linux bootloader event
+      Clean all Linux bootloader event
+      Add new ACRN efi boot event
+      Create loader.conf
+      Add default (5 seconds) boot wait time
+      Add default boot to ACRN
+      Getting latest Service OS kernel version: kernel-org.clearlinux.iot-lts2018-sos.4.19.62-73
+      Getting current Service OS kernel version: kernel-org.clearlinux.iot-lts2018-sos.4.19.13-1901141830
+      Replacing root partition uuid in acrn.conf
+      Replace with new SOS kernel in acrn.conf
+      Service OS setup done!
+      Rebooting Service OS to take effects.
+      Rebooting.
+
+#. After Service VM boots, check the ACRN status:
+
+   .. code-block:: console
+
+      $ dmesg | grep ACRN
+      [    0.000000] Hypervisor detected: ACRN
+      [    1.220887] ACRNTrace: Initialized acrn trace module with 4 cpu
+      [    1.224401] ACRN HVLog: Initialized hvlog module with 4 cpu
+
+
+Build Industry scenario Hypervisor on build machine (non-root privilege)
+========================================================================
+Follow these commands to generate the ``acrn.efi`` with industry scenario::
+
+  $ cd ~
+  $ export https_proxy=<your proxy>:<proxy port>
+  $ git clone http://github.com/projectacrn/acrn-hypervisor.git
+  $ cd acrn-hypervisor
+  $ git checkout acrn-2019w31.5-140000p
+  $ sudo vim hypervisor/arch/x86/Kconfig
+  # <Change the ACRN scenario to INDUSTRY and save>
+  choice
+         prompt "ACRN Scenario"
+         default INDUSTRY
+  $ make BOARD=kbl-nuc-i7 FIRMWARE=uefi
+  $ ls build/hypervisor/acrn.efi                    #  You will get hypervisor in this directory.
+
+Replace the acrn.efi to support industry on Service VM (root privilege)
+=======================================================================
+Now we just need to follow the steps to enable the industry scenario for Service VM::
+
+  # mount /dev/sda1 /mnt
+  # mv /mnt/EFI/acrn/acrn.efi /mnt/EFI/acrn/acrn.efi.bak
+  # scp <build machine>:~/acrn-hypervisor/build/hypervisor/acrn.efi /mnt/EFI/acrn/
+  # sync && umount /mnt
+  # reboot
+
+How to rebuild Service VM kernel from acrn-kernel tree
+******************************************************
+In this section, we will learn to build Service VM kernel and then update it onto KBL NUC.
+
+Build the Service VM kernel on build machine (non-root privilege)
+=================================================================
+Follow these commands to generate the Service VM kernel on your build machine::
+
+  $ cd ~
+  $ export https_proxy=<your proxy>:<proxy port>
+  $ git clone http://github.com/projectacrn/acrn-kernel.git
+  $ cd acrn-kernel
+  $ git checkout acrn-2019w31.5-140000p
+  $ cp kernel_config_sos .config
+  $ make menuconfig   ## Don't do anything, just save and quit to set the default config items
```

Comment:
It's confusing that we're using ``make menuconfig`` here, but previously we edited the Kconfig file (line 140).

Reply:
Now it's using ``make olddefconfig`` as the proper way :smile: